x-pack/plugins/security_solution/public/common/components/sourcerer/index.tsx

```diff
@@ -33,6 +33,7 @@ export interface SourcererComponentProps {
   scope: sourcererModel.SourcererScopeName;
 }
 
+// eslint-disable-next-line complexity
```

Comment:
maybe we can avoid this by making it a normal function instead of an arrow function

Reply:
Yes I need to remove that